src/www/ui/api/documentation/openapi.yaml

```diff
@@ -1728,6 +1728,56 @@ paths:
         default:
           $ref: '#/components/responses/defaultResponse'
 
+/file/upload/{id}/item/{ItemId}/copyrights/inactive:
```

Comment:
Please be consistent with other endpoints
```suggestion
  /uploads/{id}/items/{itemId}/copyrights:
```

Reply:
Would suggest to use a query parameter like `filter` with enum `active, inactive` rather than creating separate endpoints to get active and inactive copyrights.

Reply:
Yes sir, I will be implementing this and updating one of the PRs for this operation

Reply:
> Would suggest to use a query parameter like `filter` with enum `active, inactive` rather than creating separate endpoints to get active and inactive copyrights.

This PR's feature is implemented in #2475 using query parameters. 

@GMishx please do have a look when possible.

Reply:
So can I close this in favor of #2475 ?

Reply:
Sure sir!